internal/nodes/avi_model_nodes.go

```diff
@@ -1421,6 +1453,7 @@ func (v *AviPoolNode) CalculateCheckSum() {
 		v.LbAlgoHostHeader,
 		utils.Stringify(v.SniEnabled),
 		v.SslProfileRef,
+		v.SslKeyAndCertificateRef,
```

Comment:
I think this will impact checksum of pools in non-istio scenario also,

Reply:
updating the checksum only if reference is populated